README.md

```diff
@@ -2,7 +2,7 @@
 [![Release](https://img.shields.io/github/release/opengisch/QgisModelBaker.svg)](https://github.com/opengisch/QgisModelBaker/releases)
 [![Build Status](https://travis-ci.org/opengisch/QgisModelBaker.svg?branch=master)](https://travis-ci.com/opengisch/QgisModelBaker)
 
-![logo](branding/logo/long_logo/Long-Logo_Green_Modelbaker_RGB_QGIS.svg)
+![logo](branding/logo/long_logo/Long-Logo_Green_Modelbaker_RGB_QGIS.png)
```

Comment:
Not really a reason. Why this change is here: I changed it in the library repo because I thought that svgs cannot be displayed in the PYPI page. So I made it  the same here.

Reply:
And since it doesn't matter - I keep it. Btw. the Github App only displays PNG as well. So I thought SVG is sometimes limited.